pkgs/applications/editors/vim/plugins/non-generated/vectorcode-nvim/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  vimUtils,
+  vectorcode,
+  vimPlugins,
+}:
+let
+  inherit (vectorcode) src version;
+in
+vimUtils.buildVimPlugin {
+  inherit src version;
+
+  pname = "vectorcode.nvim";
+
+  sourceRoot = "${src.name}/plugin";
+
+  dependencies = [
+    vectorcode
+    vimPlugins.plenary-nvim
+  ];
+
+  buildInputs = [ vectorcode ];
+
+  postPatch = ''
+    ln -s ../lua .
+  '';
+
+  meta = {
+    description = "Index and navigate your code repository using vectorcode";
+    homepage = "https://github.com/Davidyz/VectorCode/blob/main/docs/neovim.md";
+    inherit (vectorcode) changelog license;
```

Comment:
```suggestion
    inherit (vectorcode.meta) changelog license;
```

Reply:
Fixed.

Also had to change the `ln -s` back to `cp -r` due to 
> ERROR: noBrokenSymlinks: the symlink /nix/store/zx3wfwd7d4nx636sqpjpd2ld1vn7brv4-vimplugin-vectorcode.nvim-0.6.10/lua points to a missing target: /nix/store/lua